course_flow/views.py

```diff
@@ -471,11 +468,11 @@ def duplicate_activity(activity: Activity, author: User) -> Activity:
         parent_activity=activity,
```

Comment:
Change to `parent_workflow=workflow` and reference the workflow object of the activity, rather than the activity itself.

Reply:
I think this is only half right. Shouldn't it be parent_workflow=activity? This field can refer to a workflow or any of its subclasses, so we might as well have a reference directly to the activity.

Reply:
Some Django inheritance managers will record a subclass model objects fields in both parent and child database tables and connect them with a OneToOneField. I believe this can result in both having the same data but different pks, if I'm correct about that I'd like the field parent_workflow to refer strictly to the parent model rather than the child's. If I'm wrong, and that isn't a thing with this manager then yeah definitely disregard changing activity to workflow.